src/frontend/packages/lib_video/package.json

```diff
@@ -6,53 +6,33 @@
   "directories": {
     "lib": "lib"
   },
-  "main": "lib/cjs/index.js",
-  "types": "lib/dts/index.d.ts",
-  "module": "lib/esm/index.js",
-  "jsnext:main": "lib/esm/index.js",
+  "main": "./src/index.ts",
+  "types": "./src/index.ts",
   "scripts": {
-    "build": "rollup -c",
-    "build:watch": "yarn build --watch",
-    "lint": "eslint src/**/*.{ts,tsx}",
-    "lint:fix": "yarn run lint -- --fix",
+    "lint": "eslint src/**/*.{ts,tsx} && tsc --noEmit",
+    "lint:fix": "yarn run lint --fix",
```

Comment:
Same here, this should not be working anymore

Reply:
https://github.com/openfun/marsha/pull/2150/commits/340a31bc806665e1e5287f18b10c47d764ebd17a